variables.tf

```diff
@@ -168,6 +168,18 @@ variable "fargate_profiles" {
   default     = {}
 }
 
+variable "worker_create_kms_key" {
+  description = "Defines if the self managed worked nodes' block devices should use a KMS Customer Managed Key instead of a KMS AWS Managed Key"
+  type        = bool
+  default     = false
+}
```

Comment:
Do we need this variable? Can we check if `worker_kms_key_arn` is not null then apply what customer passed